discord-bot/bot/api_client.py

```diff
@@ -53,14 +53,16 @@ def __init__(self, backend_url: str, api_key: str):
             TaskType.done: protocol_schema.TaskDone,
         }
 
-    async def post(self, path: str, data: dict[str, t.Any]) -> dict[str, t.Any]:
+    async def post(self, path: str, data: dict[str, t.Any]) -> Optional[dict[str, t.Any]]:
         """Make a POST request to the backend."""
         logger.debug(f"POST {self.backend_url}{path} DATA: {data}")
         response = await self.session.post(f"{self.backend_url}{path}", json=data, headers={"X-API-Key": self.api_key})
         response.raise_for_status()
```

Comment:
Thanks for improving the client interface. We probably also want to decode the exception information at the client if it is a OasstError exception .. that is something for a separate PR I will open an issue for it.